Ginger/Ginger/Actions/ActionConversion/ConversionStatusReportPage.xaml.cs

```diff
@@ -91,8 +91,9 @@ private void SetBusinessFlowConversionStatusGridView()
                 view.GridColsView = new ObservableList<GridColView>();
 
                 view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.IsSelected), WidthWeight = 5, StyleType = GridColView.eGridColStyleType.CheckBox, Header = "Select" });
-                view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.RelativeFilePath), WidthWeight = 20, ReadOnly = true, Header = "Folder" });
-                view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.BusinessFlowName), WidthWeight = 25, ReadOnly = true, Header = "Name" });
+                view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.RelativeFilePath), WidthWeight = 15, ReadOnly = true, Header = "Folder" });
+                view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.BusinessFlowName), WidthWeight = 20, ReadOnly = true, Header = "Name" });
+                view.GridColsView.Add(new GridColView() { Field = nameof(BusinessFlowToConvert.ConvertedActionsCount), WidthWeight = 13, ReadOnly = true, HorizontalAlignment= HorizontalAlignment.Center, Header = "Actions Count" });
```

Comment:
changed the header to "Converted Actions", In case of original window size (when opened) the header becomes to long so there is no space also in the column we are showing only numbers so the header seems to be big.